ui/components/app/connected-sites-list/connected-snaps.js

```diff
@@ -74,11 +74,7 @@ export default function ConnectedSnaps({ connectedSubjects }) {
             display={Display.Flex}
             alignItems={AlignItems.center}
           >
-            <SnapAvatar
-              snapId={subject.origin}
-              badgeSize={IconSize.Xs}
-              avatarSize={IconSize.Md}
-            />
+            <SnapIcon snapId={subject.origin} avatarSize={IconSize.Md} />
```

Comment:
@eriknson Can you confirm that we are dropping the "Snaps badge" for everywhere we show the icon?

Reply:
Confirming